### PR TITLE
[WIP] Refactor to match refactor-refactor in plugin repo

### PR DESCRIPTION
**Pull request by dschep**
see serverless/platform-plugin#6

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -4,18 +4,21 @@ const config = {
   local: {
     frontendUrl: 'http://localhost:3000/',
     backendUrl: 'https://a0xpn0swpd.execute-api.us-east-1.amazonaws.com/dev/',
+    logDestinationUrl: 'https://7zwgcvkt9k.execute-api.us-east-1.amazonaws.com/dev/',
```
Comment (ganeshvlrk) [inline src/config.js]: @dschep Quick question - What does the logDestinationUrl refer to?

Comment (dschep) [reply src/config.js]: The API endpoint @alexdebrie created in the platform. It returns an ARN for the cw log to send to as far as i understand

Comment (ganeshvlrk) [reply src/config.js]: The one used for prod will be different than the ones for local and dev. My understanding is that the current endpoint should be used only for local and dev.

Comment (alexdebrie) [reply src/config.js]: @ganeshvlrk I haven't put in an endpoint for prod yet as it's not deployed to prod yet. I used the same endpoint for `local`, `preview`, and `dev` in keeping w/ the core Platform backend.

File: src/credentials/getCredentials.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const platformConfig = require('../config')
+
+export default async ({ user, stageName, command, app, service, tenant }) => {
+  const body = JSON.stringify({ stageName, command, app, service })
+
+  const response = await fetch(`${platformConfig.backendUrl}tenants/${tenant}/credentials/keys`, {
+    method: 'POST',
+    body,
+    headers: {
+      Authorization: `bearer ${user.idToken}`
```
Comment (alexdebrie) [inline src/credentials/getCredentials.js]: Related comment on platform-plugin. Marking this in case we decide to change.

Comment (dschep) [reply src/credentials/getCredentials.js]: done

**Pull request merged**